sdk/core/Azure.Core/perf/PipelineDefaultBenchmark.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel.Primitives;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+
+namespace Azure.Core.Perf
+{
+    [SimpleJob(RuntimeMoniker.Net80)]
+
+    [MemoryDiagnoser]
+    public class PipelineDefaultBenchmark
```

Comment:
nit - maybe rename this to CreateAndSendWithDefaultPipeline or something similar? Just to more clearly differentiate it from PipelineBenchmark